src/main/java/org/broadinstitute/hellbender/engine/ReadsDataSource.java

```diff
@@ -167,7 +170,7 @@ public ReadsDataSource( final List<Path> samPaths, final List<Path> samIndices,
                 reader = samReaderFactory.open(samPath);
             }
             else {
-                final SamInputResource samResource = SamInputResource.of(samPath);
+                final SamInputResource samResource = SamInputResource.of(samPath, Function.identity());
```

Comment:
This change is unnecessary, since the overload that takes just a `Path` does the same thing. Can you remove?